examples/cslc_s1_sample_runconfig-v2.0.0-er.4.0.yaml

```diff
@@ -130,13 +130,17 @@ RunConfig:
                         orbit_file_path:
                             - /home/compass_user/input_dir/S1A_OPER_AUX_POEORB_OPOD_20220521T081912_V20220430T225942_20220502T005942.EOF
 
-                        # Required. The unique burst ID to process
-                        burst_id: t64_135524_iw2
+                        # The burst ID list to process
+                        burst_id: [t64_135524_iw2]
```

Comment:
For consistency's sake, lets make this field match whats in the example RTC runconfig, i.e.:

```
    # Burst to process. Empty to process all bursts
    burst_id: []
```